tests/main/auto-refresh-pre-download/task.yaml

```diff
@@ -34,6 +34,12 @@ debug: |
   cat /home/test/notif.log || true
 
 execute: |
+  err_and_exit(){
+    echo "$1"
+    tests.cleanup restore
```

Comment:
Isn't this always executed in restore-each from the suite?

Reply:
restore is called after the script ends but since there's a process still alive, it makes the test hang around until the kill-timeout elapses

Reply:
it's a quirk of how spread works internally, it runs the prepare/execute/restore scripts as separate processes and waits for each one to finish first.